ansible_collections/arista/avd/roles/eos_config_deploy_eapi/tasks/main.yml

```diff
@@ -35,4 +35,3 @@
       filename: "{{ pre_running_config_backup_filename }}"
       dir_path: "{{ pre_running_config_backup_dir }}"
   notify: "backup config"
-  tags: [provision]
```

Comment:
Should we consider adding `avd_eos_config_deploy_eapi_provision` ?

Reply:
No there is no other action in this role, so if you don't want to deploy the config, just don't run the role.